### ISSUE TITLE
accessibility: Not change state  (pressed / not pressed)  style buttons

### ISSUE THREAD

**Issue by alekssamos** [wxGlade/wxGlade#422]
In the styles left, right, center, multiline , password, ..., When pressed, the state of the button pressed and not pressed does not change for NVDA.
If completely closed (exit) and reopened wxGlade, i.e. restart, then
the last saved  state will be displayed.

**Comment by DietmarSchwertberger** [comment]
I can confirm the behaviour. I'm surprised that noone complained before.
I will look into this.

**Comment by DietmarSchwertberger** [comment]
It turned out that setting the foreground colour of checkboxes causes this problem.
So I replaced wx.BLACK with wx.NullColour.

Unfortunately there are also checkboxes which still have colours.
E.g. mutually exclusive flags are displayed in red and such which are only supported on certain wx versions are displayed in blue.

Please try out. Maybe I have to add an accessibility option for not setting checkbox colours.

**Comment by DietmarSchwertberger** [comment]
I have added an option to the bottom of the accessibility option tab.
